## frames
map_frame: "/map"
odom_frame: "/odom"

## initial pose
init_pose_x: 0.0
init_pose_y: 0.0
init_pose_a: 0.0

## Publish extra transform.
publish_extra_tf: false
extra_parent_frame_id: "/map"
extra_child_frame_id: "/origin"

## twist angular correction
use_twist_correction: true
twist_correction:
  corr_aa: 1.0
  corr_magnitude: 0.001
  alpha_filter: 0.75
  max_lin_vel: 0.5
  max_ang_vel: 0.7

## localizer parameters
localizer:
  num_particles: 5000
  min_lin_update: 0.0
  min_ang_update: 0.0
  init_stddev_x: 0.3
  init_stddev_y: 0.2
  init_stddev_a: 0.5

## odometry noise model
motion_model:
  noise_xx: 1.0
  noise_xy: 0.0
  noise_xa: 0.0
  noise_yy: 1.0
  noise_ya: 0.0
  noise_aa: 1.5
  noise_magnitude: 0.5

## likilihood field
latent_model_likelihood_field:
  uniform_hit: 0.1
  observation_sigma: 0.05

## laser beams model
laser_model:
  beam_min_distance: 0.0